roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import *
+
+
+class DockerInfo(DockerBaseClass):
+
+    def __init__(self, client, results):
+
+        super(DockerInfo, self).__init__()
+
+        self.client = client
+        self.results = results
+
+        self.results['info'] = self.get_info()
+
+    def fail(self, msg):
+        self.client.fail(msg)
+
+    def get_info(self):
+        """
+        Lookup and inspect information about the docker host.
+        :returns dictionary of host info.
+        """
+
+        return self.client.info()
+
+
+def main():
+    argument_spec = dict(
+        name=dict(type='list'),
+        )
+
+    client = AnsibleDockerClient(
+        argument_spec=argument_spec
+    )
+
+    results = dict(
+        changed=False,
+        info=[]
+    )
+
+    DockerInfo(client, results)
```

Comment:
Thanks, will update with your suggestion.

> # note: the version of AnsibleDockerClient I have installed doesn't have this method...
`client.module.info()` should work

> What makes me think, why do we even need this custom module instead of using the standard module?!

If I try to call `AnsibleDockerClient` directly in the docker_image_availability check, I get the error:
```
{"msg": "Error: Module unable to decode valid JSON on stdin.  Unable to figure out what parameters were passed", "failed": true}
```

Using `module_executor` has been the only way I have seen this work